### PR TITLE
tests: add runtime test for gce

### PR DESCRIPTION
**Pull request by aanm**
@tgraf should I rename `02-perf-gce.sh` to `09-perf-gce.sh`?

Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

**Comment by tgraf** [comment]
> @tgraf should I rename 02-perf-gce.sh to 09-perf-gce.sh?

Yes, makes sense.

File: tests/02-perf-gce.sh [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+TEST_NET="cilium"
+NETPERF_IMAGE="tgraf/nettools"
+TEST_TIME=30
+SERVER_LABEL="io.cilium.server"
+CLIENT_LABEL="io.cilium.client"
+SERVER_NAME="server"
+CLIENT_NAME="client"
+
+# Only run these tests if BENCHMARK=1 has been set
+if [ -z ${BENCHMARK} ] && [ -z ${GCE} ]; then
```
Comment (tgraf) [inline tests/02-perf-gce.sh]: I think you want `||` here

**Review by tgraf — CHANGES_REQUESTED** [review]

File: tests/02-perf-gce.sh (hunk 1 above)
Comment (aanm) [reply tests/02-perf-gce.sh]: Oh no, the doc is wrong, what I want is really to run theses tests if BENCHMARK=1 and GCE=1.

Comment (tgraf) [reply tests/02-perf-gce.sh]: You want to abort if `BENCHMARK` or `GCE` is a zero string. I think you really mean `||`

Comment (aanm) [reply tests/02-perf-gce.sh]: Ah you're right, fixed!

**Review by tgraf — APPROVED** [review]

**Pull request merged**